pkg/plugin/healthz_test.go

```diff
@@ -37,23 +40,23 @@ func TestServe(t *testing.T) {
 			desc:                   "failed to encrypt in health check",
 			setEncryptResponse:     "",
 			setEncryptError:        fmt.Errorf("failed to encrypt"),
-			expectedHTTPStatusCode: http.StatusInternalServerError,
+			expectedHTTPStatusCode: http.StatusServiceUnavailable,
```

Comment:
We now have KV calls in Status for v2. Which when errors, results in 503 instated of 500. So updating Status codes in test cases accordingly